amazon-freertos/demos/include/aws_clientcredential_keys.h.in

```diff
@@ -52,7 +52,7 @@
  */
 #ifndef keyCLIENT_CERTIFICATE_PEM
     #define keyCLIENT_CERTIFICATE_PEM    \
-    ${CLIENT_CERTIFICATE_PEM}
+    "${CLIENT_CERTIFICATE_PEM}"
```

Comment:
The quotes are not needed if we agree to enter the secret in the following format (as shown a few lines above in the comments) - without the trailing back-slash:
"-----BEGIN CERTIFICATE-----\n"
"...base64 data...\n"
"-----END CERTIFICATE-----\n"

Instructions for converting the PEM file is just add " at the beginning and \n" at the end of each line.

Reply:
In the workshop I provide a script to convert the strings and apply the changes in the code for them. Manually editing the certs to include the quotes and newlines is tedious. I update socket_startup.c as well to quickly switch between AVH and U5 build. socket_startup.c in only needed for the U5 build, but if it is ready with SSID and PASSWORD, all you have to do is switch the target and rebuild. (Correct, this is not really used in the CI/CD), but helps with KSC convenience.

Reply:
I'll just modify my script for usage in KSC. Then this PR will not be needed.